x/iro/types/params.go

```diff
@@ -18,10 +20,16 @@ var (
 	DefaultMinLiquidityPart                             = "0.4"                       // default: at least 40% goes to the liquidity pool
 	DefaultMinVestingDuration                           = 7 * 24 * time.Hour          // default: min 7 days
 	DefaultMinVestingStartTimeAfterSettlement           = 0 * time.Minute             // default: no enforced minimum by default
+
+	DefaultStandardLaunch = StandardLaunch{
+		AllocationAmount: math.NewInt(1e9).MulRaw(1e18),                                // 1B RA tokens
+		TargetRaise:      sdk.NewCoin(params.BaseDenom, math.NewInt(2e4).MulRaw(1e18)), // 20K DYM
```

Comment:
nit
```suggestion
		TargetRaise:      common.DymUint64(20_000), // 20K DYM
```

in the latest main
https://github.com/dymensionxyz/dymension/blob/caf84a4/x/common/types/constants.go#L25-L28